ocrd-import

```diff
@@ -165,104 +166,107 @@ for file in $(find -L . -type f -not -name mets.xml -not -name "*.log" | sort);
     # also, avoid . in IDs, because downstream it will confuse filename suffix detection
     base="${base//[ :.]/_}"
     if ! [[ ${base:0:1} =~ [a-zA-Z] ]]; then
-        base=f${base}
+        #base=P_${base}
+		a=0 # just do something to have a correct syntax for this 'if'
```

Comment:
non-XS-compliant page names are a problem regardless of `numpageid` or not. I don't see why this branch should be abandoned. (But if you do, `:` is the sh word for no-op.)

Reply:
Not sure what the intention was here. @stefanCCS ?

Reply:
@kba , @bertsky : My intention was to create the same result using `ocrd-import` as with using `ocrd workspace add`.
Here `page`elements looking like this
`<mets:div TYPE="page" ID="P_3871_007767869_00004">`
having an `ID`starting with `P_` (instead of starting with `f`).